src/main/java/com/greenfox/guardian/controller/GuardedEnd.java

```diff
@@ -0,0 +1,41 @@
+package com.greenfox.guardian.controller;
+
+import com.greenfox.guardian.model.Error;
+import com.greenfox.guardian.model.ErrorResponse;
+import com.greenfox.register.model.Account;
+import com.greenfox.register.model.Data;
+import com.greenfox.register.model.RequestData;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RestController
+public class GuardedEnd {
+
+  @GetMapping("/user/{id}")
+  public ResponseEntity getUserRelatedStuff(
```

Comment:
Think about how this RestController could be extended, and depending on that refactor this endpoint